drydock/patches/kustomization-resources

```diff
@@ -1,4 +1,5 @@
 - plugins/drydock/k8s/multipurpose-jobs.yml
+- plugins/drydock/k8s/patches/pdb.yml
```

Comment:
I would not include this file in the `patches` folder. This file is creating new PDB resources so I would use a path like `plugins/drydock/k8s/pdb/openedx-pdb.yml`